diag_manager/diag_yaml_format.md

```diff
@@ -43,14 +43,14 @@ diag_files:
 ### 2.1 Global Section
 The diag_yaml requires “title” and the “baseDate”.
 - The **title** is a string that labels the diag yaml.  The equivalent in the legacy diag_table would be the experiment.  It is recommended that each diag_yaml have a separate title label that is descriptive of the experiment that is using it.
-- The **basedate** is an array of 6 integers indicating the base_date in the format [year month day hour minute second].
+- The **basedate** is an array of 6 integers indicating the base_date in the format [year, month, day, hour, minute, second].
```

Comment:
Yes good idea, added that with c496f38, along with a note that spacing is not significant.

That commit also updates the rest of the test scripts to use the new base date format since they were missed originally.